app/controllers/phones_controller.rb

```diff
@@ -1,6 +1,15 @@
 # frozen_string_literal: true
 
 class PhonesController < ApplicationController
+  def create
+    phone_json = params[:phone]
+
+    phone = Phone.create(resource_id: phone_json[:resource_id], number: phone_json[:number],
```

Comment:
Should either be `create!` or you should explicitly check that the returned `phone` object was actually saved to the DB. According to the [docs](https://guides.rubyonrails.org/active_record_validations.html#valid-questionmark-and-invalid-questionmark), if you don't opt for `create!`, then you can check the `.valid?` method on the returned object.